docs/tendermint-core/metrics.md

```diff
@@ -34,21 +34,21 @@ The following metrics are available:
 | consensus_rounds                       | Gauge     | 0.21.0    |               | Number of rounds                                                       |
 | consensus_num_txs                      | Gauge     | 0.21.0    |               | Number of transactions                                                 |
 | consensus_total_txs                    | Gauge     | 0.21.0    |               | Total number of transactions committed                                 |
-| consensus_block_parts                  | counter   | on dev    | peer_id       | number of blockparts transmitted by peer                               |
-| consensus_latest_block_height          | gauge     | on dev    |               | /status sync_info number                                               |
-| consensus_fast_syncing                 | gauge     | on dev    |               | either 0 (not fast syncing) or 1 (syncing)                             |
+| consensus_block_parts                  | counter   | 0.25.0    | peer_id       | number of blockparts transmitted by peer                               |
```

Comment:
put 0.25 here as i couldn't find in the changelog when they were added. I can spend sometime to find the exact releases if people want it but will take a bit of time